.gitlab-ci.yml

```diff
@@ -0,0 +1,47 @@
+include:
+  local: templates/v6.yml
+
+.install:
+  script: |
+    apt-get -qq update
+    DEBIAN_FRONTEND=noninteractive apt-get -qqy -o=Dpkg::Use-Pty=0 install wget
+    eval "$(bash install_julia.sh $JULIA)"
+    hash julia 2> /dev/null
+
+Install Julia 1.0:
+  extends: .install
+  variables:
+    JULIA: '1.0'
+
+Install Julia nightly:
+  extends: .install
+  variables:
+    JULIA: nightly
+
+Install Julia source:
+  extends: .install
+  variables:
+    JULIA: source
+  tags:
+    - intel
```

Comment:
why the intel tag?

Reply:
I wasn't sure if runners assigned to untagged jobs would be efficient for building Julia. Is it a safe assumption that untagged == decent CPU?

Reply:
I think this also raises the similar question of how we should be tagging jobs that don't require a GPU. Should the standard be untagged, should there be a `cpu` tag, or should that tag be `intel`?

Reply:
The `intel` runner is for oneAPI.jl, and there's only a single system servicing that tag. Maybe we could just use the shared runners here? We should probably have a global `image: ubuntu` then.

Reply:
That seems reasonable, in that case we should remove the recommendation to disable shared runners.